superset-frontend/src/components/EditableTitle/index.tsx

```diff
@@ -70,6 +70,13 @@ export default function EditableTitle({
   useEffect(() => {
     if (isEditing) {
       contentRef.current.focus();
+      // move cursor and scroll to the end
+      if (contentRef.current.setSelectionRange) {
+        const { length } = contentRef.current.value;
+        contentRef.current.setSelectionRange(length, length);
+        contentRef.current.scrollLeft = contentRef.current.scrollWidth;
+        contentRef.current.scrollTop = contentRef.current.scrollHeight;
```

Comment:
Not sure if this is needed, but it probably doesn't hurt, either

Reply:
Cool. Here I think it need to scroll to the bottom for the textarea.

Reply:
> Cool. Here I think it need to scroll to the bottom for the textarea.

Right you are! 👍